### ISSUE TITLE
Architecture bake-off: DualStream replaces the identity top-skip, so its result is not attributable

### ISSUE THREAD

**Issue by Polichinel** [views-platform/views-hydranet#366]
Found by `/code-review max` on #351. **Routed out of Epic #353** — a bake-off attribution problem.
Owner: `reports/2026-08-24_architecture_bakeoff_dossier`.

## The defect

The incumbent's top-skip is the identity (`HydraBNrecurrentUnet_06_LSTM4.py:547`):

```python
return torch.cat([e0s, coords], 1) if coords is not None else e0s
```

`DualStream._topskip` (`dual_stream.py:49-55`) **replaces** it with a randomly-initialised
1×1 conv + BatchNorm + ReLU:

```python
self.fuse    = nn.Conv2d(base + width, base, 1, bias=False)
self.bn_fuse = nn.BatchNorm2d(base)
...
out = F.relu(self.bn_fuse(self.fuse(torch.cat([e0s, s], 1))))
```

It is **not zero-initialised** and **not residual** — `e0s` is never added back. So **even with the
parallel stream emitting exactly 0.0, the signal changes**: the path becomes `relu(bn(W·e0s)) ≠ e0s`.

Measured with every `self.stream` parameter zeroed and its BN reset: correlation with `e0s` is
**r = 0.0084**, and **50.5%** of the skip is zeroed by the fuse ReLU.

## Why it matters

The incumbent's only full-resolution path from input to the six decision layers is destroyed and
re-mixed at random **before training starts**. A DualStream loss is therefore **not attributable** to
*"a parallel full-resolution stream does not help"* — it is indistinguishable from *"replacing the
identity skip with an untrained 1×1+BN+ReLU hurt"*.

The dossier registers only the **parameter-count** confound for this arm and states the fusion leaves
*"the six decoder paths untouched"* — true of the shapes, not of the signal.

**The sibling arm gets this right.** `FiLMSkip` (`dynamic_skip.py:83-84`) zero-initialises so it
starts at the incumbent. `DynamicTopSkip` concatenates `e0s` verbatim. DualStream is the outlier.

The mechanism test does not catch it: `test_candidate_mechanisms.py:125-140` perturbs
`m.stream.parameters()` and asserts the output moved — it never asserts that a **zero** stream
reproduces the incumbent.

## Work

- [ ] Make the arm start at the incumbent: residual (`e0s + relu(bn_fuse(fuse(...)))`) or a zero-init
      fuse, matching the FiLMSkip discipline the dossier praises
- [ ] A test that a zeroed stream reproduces the incumbent's top-skip **byte-identically**
- [ ] Note in the dossier that any DualStream result measured before this fix carries the confound

## Acceptance criteria

- [ ] With the parallel stream at zero, `_topskip` output equals the incumbent's
- [ ] The test fails when the fix is reverted
- [ ] The dossier's confound list names this alongside the parameter-count one
